sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/aspnethostedservice/Azure.Messaging.EventHubs.Samples.Processor.HostedService.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <RequiredTargetFrameworks>net7.0;net6.0</RequiredTargetFrameworks>
```

```suggestion
    <RequiredTargetFrameworks>net8.0;net6.0</RequiredTargetFrameworks>
```
We may as well switch to `net8.0`, as it's now GA.

---

@jsquire I've left this as net7.0 as I had issues with the SDK. I believe globals.json needs updating first? https://github.com/Azure/azure-sdk-for-net/blob/main/global.json

---

That's my bad.  I'm getting ahead of myself and forgetting that we haven't updated the SDK version yet.  Leaving it as-is makes sense.